pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -199,6 +197,21 @@ type PipeSpec struct {
 	Key string `json:"key"`
 }
 
+// KudoOperatorSpec describes a how a KUDO operator is installed
```

Comment:
```suggestion
// KudoOperatorSpec describes how a KUDO operator is installed
```
Also nitpick: I don't see how this describes _how_ a KUDO operator is installed, but _that_ a KUDO operator should be installed using these specs.